CONTRIBUTING.md

```diff
@@ -9,13 +9,29 @@ contributor license agreement.
 
 ## Submitting Changes
 
-Submit a pull request rebased on top of master
+Submit a pull request rebased on top of master.
 
- * Include a descriptive commit message.
- * Changes contributed via pull request should focus on a single issue at a time.
+Workflow:
 
-At this point you're waiting on us. We like to at least comment on pull requests within one week
+1. **Fork repo on GitHub:** Visit the [`stellar/stellar-core`](https://github.com/stellar/stellar-core) repository and click on the `Fork` button to create a copy of the repository under your GitHub account.
```

Comment:
I think it would be a bit nicer to keep this section focused on core-specific instructions, just to give contributors an idea on how to get started. So perhaps we don't need detailed git commands, and could simply mention that we use forked repos (as opposed to feature branches) for pull request flow, and generally keep our commit history tidy and up-to-date with master.

Reply:
Fair, I think we would benefit from an explicit workflow but I agree it is verbose. I could modify this section to just say:

## Submitting Changes

Fork the `stellar/stellar-core` repo on github and submit a pull request from a branch in your fork. 
Key things to keep in mind:
* Keep your branch up-to-date by rebasing on `stellar/stellar-core` `master`
* Ensure your branch focuses on a single issue at a time
* Before merging, squash your commits and ensure you have a descriptive commit message